src/dune_rules/simple_rules.ml

```diff
@@ -216,17 +217,27 @@ let copy_files sctx ~dir ~expander ~src_dir (def : Copy_files.t) =
         Path.build file_dst)
   in
   let+ () =
-    Memo.Option.iter def.alias ~f:(fun alias ->
+    Memo.Option.iter alias ~f:(fun alias ->
         let alias = Alias.make alias ~dir in
         Rules.Produce.Alias.add_deps alias (Action_builder.path_set targets))
   in
   targets
 
+let copy_files sctx ~dir ~expander ~src_dir
+    ({ add_line_directive; alias; mode; files; syntax_version; _ } :
+      Copy_files.t) =
+  copy_files' sctx ~dir ~expander ~src_dir ~add_line_directive ~alias ~mode
+    ~files ~syntax_version
+
 let copy_files sctx ~dir ~expander ~src_dir (def : Copy_files.t) =
   Expander.eval_blang expander def.enabled_if >>= function
   | true -> copy_files sctx ~dir ~expander ~src_dir def
   | false -> Memo.return Path.Set.empty
 
+let melange_runtime_deps sctx ~dir ~expander ~src_dir files =
+  copy_files' sctx ~dir ~expander ~src_dir ~add_line_directive:false ~alias:None
```

Comment:
Do you need the glob here? I don't see the glob functionality being tested anywhere.

Reply:
I think the glob will be very useful to have, yes. I will add some tests to exercise the functionality.

While checking this, I realized a couple of things:
- we want to allow multiple globs per field, e.g. `(melange.runtime_deps assets/*.png fonts/file.ttf)`
- the current implementation is broken, the tests where only working by chance

I will update the PR with some improvements and fixes.